src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruitName;
+    private int amount;
+
+    public Operation getOperationByLetter(String letter) {
```

Comment:
Just wondering if it is correct to make `Operation` in the same file as `FruitTransaction`? I see it is used not only inside enclosing class, so probable we can extract `Operation` to the separate file?